experiments/calibration/models/snowy_land.jl

```diff
@@ -0,0 +1,193 @@
+# # Global run of land model
+
+# The code sets up and runs ClimaLand v1, which
+# includes soil, canopy, and snow, on a spherical domain,
+# using ERA5 data as forcing. In this simulation, we have
+# turned lateral flow off because horizontal boundary conditions and the
+# land/sea mask are not yet supported by ClimaCore.
+
+# Simulation Setup
+# Number of spatial elements: 101 in horizontal, 15 in vertical
+# Soil depth: 50 m
+# Simulation duration: 730 d
+# Timestep: 450 s
+# Timestepper: ARS111
+# Fixed number of iterations: 3
+# Jacobian update: every new Newton iteration
+# Atmos forcing update: every 3 hours
+
+function setup_model(
```

Comment:
I recently ran into a compiler bug on Clima and locally with this function. It had to be changed to 
function setup_model(::Type{FT},...) where {FT}

Reply:
Dennis can better explain, but it was something about this type being undefined when the function is being compiled and so something downstream inside the function broke when it used {FT} in its constructor